hbp_knowledge/receptors/wang2016.bel

```diff
@@ -61,27 +61,37 @@ SET Citation = {"PubMed", "26837465"}
 SET Evidence = "In the hippocampal and cortical tissues of all AD cases examined, neuronal staining for ERα  was a prominent finding,\
 both with a weak cytoplasmic stain, but often specifically with a fibrillar appearance suggesting ERα  was in neurofibrillary tangles\
 (Fig. 1A), also shown at a higher magnification (Fig. 1B)."
-SET MeSHAnatomy = {"Cerebral Cortex", "Hippocampus"}
 SET Confidence = "Medium"
+SET MeSHAnatomy = {"Cerebral Cortex", "Hippocampus"}
 p(HGNC:ESR1) -- complex(GO:"neurofibrillary tangle")
+composite(p(GO:"neurofibrillary tangle"),p(HGNC:ESR1)) -> complex(p(GO:"neurofibrillary tangle"),p(HGNC:ESR1))
```

Comment:
```suggestion

```
no i would not add this, because a colocalization does not necessary means that they form a complex/interact
therefore the association statement was added